packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -55,20 +56,21 @@ export default function Classifier({
 
   useEffect(function onURLChange() {
     const { workflows } = classifierStore
-    if (workflowID && workflowSnapshot) {
+    if (workflowID) {
       console.log('starting new subject queue', { workflowID, subjectSetID, subjectID })
       workflows.setResources([workflowSnapshot])
```

Comment:
Is there any scenario where `onURLChange()` will be triggered, but `worfklowSnapshot` is undefined or null? Does the WorkflowStore care if the snapshot happens to be undefined or null?

Reply:
Sure. The snapshot will be undefined while we’re waiting for a workflow to load, but I think the container component waits until the workflow has loaded before mounting the classifier. 

The tests don’t include the container at the moment. The mocks in the `Classifier` should match what we expect the container to pass down but it would probably be more robust to include `ClassifierContainer` in the tests, and test against a mock Panoptes API.

Reply:
Without looking at the code for the store model, I’d guess that passing null here would throw an error. MobX State Tree is strict about [types](https://mobx-state-tree.js.org/overview/types) and I don’t know if the classifier models allow null or undefined for Panoptes resources, like workflows.

Reply:
Sounds good, thank you for the clarification 👍